### PR TITLE
Add blog post for Bazel 4.1.0

### PR DESCRIPTION
**Pull request by philwo**

### CODE REVIEW

File: _posts/2021-05-21-bazel-4-1.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+* Removed a non-actionable "Targets were missing from graph" warning printed by `cquery` related to toolchains. ([#11993](https://github.com/bazelbuild/bazel/issues/11993))
+
+## Coverage
+
+* Fixed `bazel coverage` with C++ to work with Remote Execution. ([#13193](https://github.com/bazelbuild/bazel/issues/13193), [#13232](https://github.com/bazelbuild/bazel/issues/13232))
+
+## External dependencies
+
+* Added SHA-1 to subresource integrity format for download() checksums. ([#12777](https://github.com/bazelbuild/bazel/issues/12777))
```
Comment (meisterT) [inline _posts/2021-05-21-bazel-4-1.md:31]: nit: put download() in backticks

**Review by meisterT — APPROVED** [review]

**Comment by philwo** [comment]
@hicksjoseph @floriographygoth This should essentially be the same text that you already reviewed in the draft release notes Google Doc, but please have a look and LGTM just to make sure :)

File: _posts/2021-05-21-bazel-4-1.md (hunk 1 above)
Comment (philwo) [reply _posts/2021-05-21-bazel-4-1.md]: Fixed.

**Comment by hicksjoseph** [comment]
I can see that it is merged, but LGTM anyway.

**Pull request merged**